src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -583,6 +583,7 @@ paths:
         uri: '${base_uri}/eros/{eroId}/anonymous-elector-documents'
         requestParameters:
           integration.request.path.eroId: method.request.path.eroId
+          integration.request.querystring.applicationId: method.request.querystring.applicationId
```

Comment:
Reference : `https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-method-settings-execution-console.html` for `query` string `applicationId`. 

We might need to do the same for other GET endpoints

Reply:
I think if the query parameter name is the same in the AWS API GW and the endpoint the request is forwarded the passthrough behaviour means no explicit mapping is required.